client/less/main.less

```diff
@@ -1071,6 +1071,21 @@ code {
       background-color: #666;
     }
   }
+  .hamburger-dropdown {
+    .dropdown .dropdown-menu,
+    .open #nav-Community-dropdown {
+      background-color: @gray;
+      a {
+        color: @night-text-color;
+      }
+    }
+    a:focus,
+    a:hover,
+    a:active {
+      background-color: #666 !important;
```

Comment:
I'm not sure about the background colour change on the navbar item. I think it's fine, but it's a little bit distracting to see it change when I move my mouse off of it onto the dropdown items.